h/app.py

```diff
@@ -66,6 +67,8 @@ def create_app(global_config, **settings):
         config.add_view(api_view, name='api', decorator=strip_vhm,
                         route_name='index')
 
+    config.include('.api_client')
```

Comment:
Nitpick: would you mind moving this up above the feature checks so that features can all be together?